v20.1/admin-ui-databases-page.md

```diff
@@ -7,35 +7,53 @@ toc: true
 On a secure cluster, this area of the Admin UI can only be accessed by an `admin` user. See [Admin UI access](admin-ui-overview.html#admin-ui-access).
 {{site.data.alerts.end}}
 
-The **Databases** page of the Admin UI provides details of the databases configured, the tables in each database, and the grants assigned to each user. To view these details, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) and then click **Databases** on the left-hand navigation bar.
+The **Databases** page of the Admin UI provides details of the databases configured, the tables in each database, and the grants assigned to each user. 
 
+To view this page, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) and click **Databases** in the left-hand navigation.
 
 ## Tables view
 
-The **Tables** view shows details of the system table as well as the tables in your databases. To view these details, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) and then select **Databases** from the left-hand navigation bar.
+The **Tables** view shows details of the system table as well as the tables in your databases.
+
+To view [table details](#table-details), click on a table name.
 
 <img src="{{ 'images/v20.1/admin_ui_database_tables_view.png' | relative_url }}" alt="CockroachDB Admin UI Database Tables View" style="border:1px solid #eee;max-width:100%" />
 
-The following details are displayed for each table:
+The following are displayed for each table:
 
-Metric | Description
+Parameter | Description
 --------|----
 Table Name | The name of the table.
-Size | Approximate total disk size of the table across all replicas.
+Size | Approximate disk size of all replicas of this table on the cluster.
 Ranges | The number of ranges in the table.
 \# of Columns | The number of columns in the table.
 \# of Indices | The number of indices for the table.
 
+### Table details
+
+Click any table name in [Tables](#tables-view) view to display details for that table.
+
+<img src="{{ 'images/v20.1/admin_ui_database_tables_details.png' | relative_url }}" alt="CockroachDB Admin UI Database Tables View" style="border:1px solid #eee;max-width:100%" />
+
+- **Overview** displays the SQL statements used to create and define the table, as well as partitioning info and zone configurations. In addition, the following metrics are displayed:
+	- **Size** is the approximate disk size of all replicas of this table on the cluster.
+	- **Ranges** is the number of [ranges](architecture/overview.html#terms) in this table.
+	- **Replicas** is the number of [replicas](architecture/replication-layer.html) of this table on the cluster.
+<!-- - **Statements** displays SQL statements that have been run against the table. 
```

Comment:
This commented out part is showing in the build output.  Suggest deleting it altogether (git history will still have it if we need it)